data/hypha.css

```diff
@@ -597,6 +597,12 @@ input {
  background-color:  var(--menulinkhoverback);
 }
 
+div.comment-subject {
+  font-style: italic;
```

Comment:
You emphasized this title in three ways:

- make it larger
_ make it bolder
- add italic

I would prefer just to make it larger, leave out the others.